movemap.h

```diff
@@ -8,20 +8,31 @@
 #include "element.h"
 
 /**
- * @brief 
+ * Contains information about a sequence of moves in a convenient format
+ * used in the move function of Board.
+ * 
+ * Holds two maps, one for inputs of the move and one for the outputs.
+ * Each map has keys for the elements the spheres the gems are moving from/to,
+ * and values containing map of each gem type and the number of those gems involved
+ * in the move.
```

Comment:
This is a good description, but I think it needs a bit more. Why is it good to have two maps? Can we explain that here?